### ISSUE TITLE
AOI（Area of interest）covered two frame do SBAS

### ISSUE THREAD

**Issue by Yaoshangheng** [gmtsar/gmtsar#666]
Hi，professor
I wonder if GMTSAR could do this?
My AOI is covered across two frame for example both T465 and T470 which has the same Path.
I was going to do SBAS analysis，What should I do for using GMTSAR？

![frame](https://user-images.githubusercontent.com/20898434/221348580-a37cad43-af23-4768-90ef-a78fc325ac16.jpg)

Best wish！

**Comment by Xiaohua-Eric-Xu** [comment]
Try create_frame_tops.csh to stitch two frames together and process this pair with p2p_S1_TOPS_Frame.csh.

**Comment by Yaoshangheng** [comment]
Yes，professor，I was going to do time-series.Such as my AOI spans two frames.My data range span 5 years.
Should I do SBAS process at each frame separately? Or re-assemble frames together and do SBAS process.

Sorry for bothering you.

**Comment by Xiaohua-Eric-Xu** [comment]
OK then you should follow the document. I saw that you have 4 scenes in the search so thought you are processing a pair.

**Comment by igps-ftk** [comment]
> Yes，professor，I was going to do time-series.Such as my AOI spans two frames.My data range span 5 years. Should I do SBAS process at each frame separately? Or re-assemble frames together and do SBAS process.
> 
> Sorry for bothering you.

I wrote a few scripts to facilitate processing Sentinel-1 data with GMTSAR. Please check the tutorial for more.
https://github.com/igps-ftk/iGPS/blob/master/doc/InSAR.Processing.with.GMTSAR.and.iGPS-20230228.pdf
The latest codes can be found at https://github.com/igps-ftk/iGPS.

Cheers,
Yunfeng Tian

**Comment by Xiongwei789** [comment]
> > Yes，professor，I was going to do time-series.Such as my AOI spans two frames.My data range span 5 years. Should I do SBAS process at each frame separately? Or re-assemble frames together and do SBAS process.
> > Sorry for bothering you.
> 
> I wrote a few scripts to facilitate processing Sentinel-1 data with GMTSAR. Please check the tutorial for more. https://github.com/igps-ftk/iGPS/blob/master/doc/InSAR.Processing.with.GMTSAR.and.iGPS-20230228.pdf The latest codes can be found at https://github.com/igps-ftk/iGPS.
> 
> Cheers, Yunfeng Tian

Hi, 
Thank you for sharing. I read your scripts of GACOS corrections. You remove atmospheric effects from unwrapped interferogram (unwrap_mask.grd). However, there is a script that performs GACOS corrections before unwarping (phasefilt.grd, https://github.com/esrivas17/gacos_correction_for_GMTSAR/blob/master/operation.cshphasefilt.grd).
I don't know which is reasonable.

**Comment by igps-ftk** [comment]
> Hi, Thank you for sharing. I read your scripts of GACOS corrections. ...

Yeah, this is a bit off the topic. Anyway, I was under the impression that GACOS correction is in the unwrapped manner.  I think it makes no difference if there is no phase ambiguity. But maybe I am wrong.

Cheers,
Yunfeng Tian

**Comment by Xiongwei789** [comment]
> Yeah, this is a bit off the topic. Anyway, I was under the impression that GACOS correction is in the unwrapped manner. I think it makes no difference if there is no phase ambiguity. But maybe I am wrong.
Yes, you should be right. I checked examples on the GACOS website. GOCAS correction is indeed carried out in unwrapped phase.

**Comment by Xiongwei789** [comment]
> Yeah, this is a bit off the topic. Anyway, I was under the impression that GACOS correction is in the unwrapped manner. I think it makes no difference if there is no phase ambiguity. But maybe I am wrong.

Yes, you should be right. I checked examples on the GACOS website. GOCAS correction is indeed carried out in unwrapped phase.

**Comment by Yaoshangheng** [comment]
> 

Thanks for sharing.
I'm really grateful to you.

Best wishes!
Yuan Yao
